README.md

```diff
@@ -62,7 +62,22 @@ Visit [RBAC permissions for running the tasks](docs/tasks-rbac-permissions.md) i
 
 ## Examples
 
-#### [Unit Tester Pipeline](examples/pipelines/unit-tester) 
+#### [Windows bios Installer Pipeline](pipelines/windows-bios-installer)
+
+Downloads a Windows ISO file into a PVC and automatically installs Windows in BIOS mode by using a custom Answer file into a new base DV.
+Supported Windows versions: Windows10
+
+#### [Windows efi Installer Pipeline](pipelines/windows-efi-installer)
+
+Downloads a Windows ISO file into a PVC and automatically installs Windows 11, or 2k22 with EFI enabled by using a custom Answer file into a new base DV.
+Supported Windows versions: Windows11, Windows Server 2k22
+
+#### [Windows Customize Pipeline](pipelines/windows-customize)
+
+Applies customizations to an existing Windows 10, 11, 2k22 installation by using a custom Answer file and creates a new base DV.
+Supported Windows versions: Windows10, Windows11, Windows Server 2k22
+
+#### [Unit Tester Pipeline](examples/pipelines/unit-tester) - Unmaintained
```

Comment:
Do we want to frame these three pipelines as unmaintained? Is there much to maintain? If somebody has a problem they can open an issue about it?

Reply:
these pipelines are currently not working. I don't have time to maintain them. For sure they can open issue about them

Reply:
ACK